dispatcher_plugin_nb2workflow/dataserver_dispatcher.py

```diff
@@ -98,6 +98,44 @@ def test_has_input_products(self, instrument, logger=None):
         query_out.set_done('input products check skipped')
         return query_out, []
 
+    def get_progress_run(self, **kwargs):
```

Comment:
Why `**kwargs` and not an explicit list of arguments?

Reply:
I initially wanted to make it flexible with the args that will be passed from the dispatcher, but those will always be the same. 

So yes, no need for `kwargs`